src/main/java/com/fcgl/madrid/search/service/UserSearchService.java

```diff
@@ -1,18 +1,69 @@
 package com.fcgl.madrid.search.service;
 
-import com.fcgl.madrid.search.dataModel.Category;
+import com.fcgl.madrid.search.dataModel.UserSearch;
 import com.fcgl.madrid.search.payload.InternalStatus;
+import com.fcgl.madrid.search.payload.request.UserSearchRequest;
 import com.fcgl.madrid.search.payload.response.Response;
-import com.fcgl.madrid.search.repository.CategoryRepository;
+import com.fcgl.madrid.search.repository.UserSearchRepository;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.SpringApplication;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Service;
 
+import java.util.ArrayList;
+import java.util.Date;
 import java.util.List;
 
 @Service
 public class UserSearchService {
     //TODO: Add necessary functions
+
+    private UserSearchRepository repo;
```

Comment:
Let's name this `userSearchRepo` in case we need to add another repo to this service later on